src/rtutils.c

```diff
@@ -1072,6 +1078,9 @@ static size_t jl_static_show_x_(JL_STREAM *out, jl_value_t *v, jl_datatype_t *vt
         char *sn = jl_symbol_name(sym);
 
         jl_sym_t *globname;
+        if (!is_globname_binding(v, dv)) {
```

Comment:
If mt can be null, should this check go in the else if above instead of after we dereferenced mt?

Reply:
Yeah, might as well.